frontend/js/behaviors/customTourBuilder/customTourBuilder.js

```diff
@@ -0,0 +1,25 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import CustomTourBuilder from 'custom-tour-builder';
+
+export default function customTourBuilder(container) {
+  this.init = function () {
+    ReactDOM.render(
+      <CustomTourBuilder
+        initMessage="Custom Tour Builder will render here!"
+      />,
+      container,
+      () => {
+        // This is informed by where I think the header scroll hiding is happening
+        // See: frontend/js/functions/core/setScrollDirection.js
+        // Catch if the scroll is below the threshold and remove the class
+        if (window.scrollY < 100) {
+          document.documentElement.classList.remove('s-header-hide');
+        }
```

Comment:
I assume the React page will use the same sticky header logic already defined on the website. Why does this need to be specified here?

Reply:
That's correct. There's a bit more discussion of this [here](https://github.com/Cogapp/artic.edu/pull/46).

Hannah noticed while we were reviewing this that if the page was reloaded it looked like the header doesn't always appear, on closer inspection it was being hidden after tour builder loaded in, possibly some kind of race condition or to do with the page height changing, but this is to catch the case where it definitely should be visible.

Reply:
Ok, I'll flag this to review on our side after delivery.